### DISCUSSION TITLE
file:// url link in GTK3 app - qutebrowser or file browser is not launched

### DISCUSSION THREAD

**Discussion by gadefox** [qutebrowser/qutebrowser#7898] [Q&A]
`gnome-disks` is an application using `GTK3`. When I mount the disk, the path is displayed as an url e.g. `<a href="file:///media/fox">/media/fox</a>`. The link does not open file manager, any idea??

`xdg-mime query filetype /media/fox` returns `inode/directory`
`xdg-mime query default inode/directory` returns `nnn.desktop`
`xdg-open /media/fox` opens `nnn` terminal based file manager

No desktop environment is installed e.g. `GNOME`, `Mate` etc..

**Comment by The-Compiler** [comment]
I'm confused - none of this seems to be related to qutebrowser?

**Comment by gadefox** [comment]
Mea culpa, this seems to be a `GTK` issue - feel free to delete the thread.
